--- origsrc/anjuta-extras-2.28.0/configure.ac	2009-09-22 04:02:04.000000000 -0500
+++ src/anjuta-extras-2.28.0/configure.ac	2009-11-05 04:42:41.742890600 -0600
@@ -99,9 +99,9 @@ else
 	valgrind="yes"
 fi
 
-AC_CHECK_LIB(bfd, bfd_get_error, [PLUGIN_VALGRIND_LIBS="-Wl,-Bstatic -lbfd -liberty -Wl,-Bdynamic -lz"],
+AC_CHECK_LIB(bfd, bfd_get_error, [PLUGIN_VALGRIND_LIBS="-Wl,-Bstatic,-lbfd,-liberty,-Bdynamic,-lz,\$(INTLLIBS)"],
    valgrind="no",
-  [-liberty -lz])
+  [-liberty -lz $INTLLIBS])
 
 AC_SUBST(PLUGIN_VALGRIND_LIBS)
 AM_CONDITIONAL(HAVE_PLUGIN_VALGRIND, [test x$valgrind = xyes])
